tests/hook/test_init_hook.py

```diff
@@ -70,7 +71,7 @@ def test_validate_plugin_choice_greater_than_choice():
 
 
 def test_validate_plugin_choice_valid():
-    choices = ["1", PROMPT, "2"]
+    choices = ["python38", PROMPT, "pyton39"]
```

Comment:
Last choice should be "python39" -- missing an 'h'

Reply:
fixed typo